lib/convict.js

```diff
@@ -140,12 +141,13 @@ function normalizeSchema (name, node, props, fullName, env, argv) {
     if (BUILT_INS.indexOf(format) >= 0) {
       // if the format property is a built-in JavaScript constructor,
       // assert that the value is of that type
+      var Format = typeof format == 'string' ? eval(format) : format;
```

Comment:
Yeah, I prefer that. I wanted to avoid creating a separate object for the sake of a mapping, but it's safer